docs/modules/operation/pages/deep-dive/performance-data-collection/snmp-index.adoc

```diff
@@ -65,11 +67,13 @@ Here is the data to collect for the hrStorageIndex resource type:
 [[system-def]]
 == Add the group to a system definition
 
-In datacollection-config.xml, OpenNMS needs to be able to map a remote SNMP agent's system object ID to a set of data collection groups so it knows what it should gather for different types of SNMP agents.
+{page-component-title} needs to be able to map a remote SNMP agent's system object ID to a set of data collection groups in `datacollection-config.xml`.
+This ensures that {page-component-title} knows what should be gathered for different types of SNMP agents.
```

Comment:
It looks like this information is gathered by the WsManCollector - since that's outside the scope of the Quick Start, I think just saying that OpenNMS collects the data should be fine.